### ISSUE TITLE
/Library/Ruby/Gems/1.8/gems/roundsman-0.1.0/lib/roundsman/capistrano.rb:242:in `match': can't convert Symbol into String (TypeError) 

### ISSUE THREAD

**Issue by vjpr** [iain/roundsman#6]
Whenever I try to run `roundsman.run_list` or `roundsman:configuration` I get this message.

**Comment by vjpr** [comment]
Adding `to_s` to `key` here fixed it.

```
next if fetch(:filter_sensitive_settings).find { |regex| regex.match(key) }
```

**Comment by vjpr** [comment]
FYI

```
node$ ruby -v
ruby 1.8.7 (2012-02-08 patchlevel 358) [universal-darwin12.0]
```

**Comment by Arie** [comment]
Fixed in 9ae1a01d963207bc904a3f306a3ffc924b753f94
